src/components/kvittering/arbeidstaker.tsx

```diff
@@ -0,0 +1,118 @@
+import dayjs from 'dayjs'
+import AlertStripe, { AlertStripeInfo, AlertStripeSuksess } from 'nav-frontend-alertstriper'
+import Lenke from 'nav-frontend-lenker'
+import { Element, Normaltekst, Undertekst, Undertittel } from 'nav-frontend-typografi'
+import React, { useEffect, useState } from 'react'
+
+import { useAppStore } from '../../data/stores/app-store'
+import { tekst } from '../../utils/tekster'
+import Avkrysset from '../oppsummering/utdrag/avkrysset'
+import Utvidbar from '../utvidbar/utvidbar'
+import Vis from '../vis'
+import { Mottaker } from './innhold/kvittering-status'
+
+const Arbeidstaker = () => {
+    const { valgtSoknad } = useAppStore()
+    const [ tilArbNavn, setTilArbNavn ] = useState<string>()
+    const [ tilOrg, setTilOrg ] = useState<string>()
+    const [ tilNavDato, setTilNavDato ] = useState<string>()
+    const [ tilArbDato, setTilArbDato ] = useState<string>()
+
+    useEffect(() => {
+        opprettDatoer()
+        // eslint-disable-next-line
+    }, [])
+
+    const opprettDatoer = () => {
+        const sendtTilNav = valgtSoknad?.sendtTilNAVDato
+        if (sendtTilNav) {
+            const datoNav = dayjs(sendtTilNav).format('dddd D. MMM, kl hh:mm')
+            setTilNavDato(datoNav.charAt(0).toUpperCase() + datoNav.slice(1))
+        }
+
+        const sendtTilArb = valgtSoknad?.sendtTilArbeidsgiverDato
+        if (sendtTilArb) {
+            const datoArb = dayjs(sendtTilArb).format('dddd D. MMM, kl hh:mm')
+            setTilArbDato(datoArb.charAt(0).toUpperCase() + datoArb.slice(1))
+            setTilArbNavn(valgtSoknad?.arbeidsgiver?.navn ? valgtSoknad?.arbeidsgiver?.navn : Mottaker.ARBEIDSGIVER)
+            setTilOrg(valgtSoknad?.arbeidsgiver?.orgnummer ? `(Org.nr. ${valgtSoknad.arbeidsgiver.orgnummer})` : '')
+        }
+    }
+    //TODO: legge til vedtaksløsningen
+    const over16dager = dayjs(valgtSoknad!.tom).diff(dayjs(valgtSoknad!.fom), 'day') > 16
```

Comment:
Dette må hentes fra finnMottakerAvSøknad. Ref snakk i planleggingsmøtet og min kommentar i trello. Det er litt mer komplisert enn over og under 16 dager i en søknad. Det går evnt ann å merge det som det er og så fikse det for seg selv.